lib/github/index.js

```diff
@@ -17,7 +17,11 @@ module.exports = (robot) => {
 
   robot.on('push', middleware(push));
 
-  robot.on(['pull_request.opened', 'pull_request.closed', 'pull_request.reopened', 'pull_request.edited'], middleware(pullRequest));
+  if (process.env.NO_BACKGROUND_WEBHOOKS === 'true') {
```

Comment:
I'm going to keep both paths in production code, but we can quickly back out my worker changes if we see something bad happen.